app/views/api/v3/analytics/user_analytics/show.html.erb

```diff
@@ -236,6 +247,73 @@
     </div>
 </div>
 
+<%= render "drug_stock_form" %>
+
+<div id="drug-stock-submitted" class="progress-body" style="display: none;">
+  <div class="progress-contents">
+      <a href="#" onclick="closeWindow('drug-stock-submitted', 'drug-stock-form'); return false" class="help-back">
+          <%= inline_svg('icon_back.svg') %>
+      </a>
+
+      <div class="card" style="margin-top: 100px; text-align: center;">
+        <div style="padding: 40px 0 20px 0;"><%= inline_svg('check.svg') %></div>
+        <h2>Submitted!</h2>
+        <p>We automatically estimated how many days of drug stock remain, based on the number of patients registered at your facility.</p>
+      </div>
+
+      <div class="card">
+        <h3 style="margin-bottom: 8px;">ARB tablets</h3>
+        <span class="text-green">245 days of drug stock</span>
+
+        <div class="card-row">
+          <span class="light float-right">1456 tablets</span>
+          <h6>Amlodipine 5 mg</h6>
+        </div>
+        <div class="card-row">
+          <span class="light float-right">1456 tablets</span>
+          <h6>Amlodipine 10 mg</h6>
+        </div>
+      </div>
+
+      <div class="card">
+        <h3 style="margin-bottom: 8px;">CCB tablets</h3>
+        <span class="text-red">1 day of drug stock</span>
+
+        <div class="card-row">
+          <span class="light float-right">&#8212;</span>
+          <h6>Losartan 50 mg</h6>
+        </div>
+        <div class="card-row">
+          <span class="light float-right">10 tablets</span>
+          <h6>Telmisartan 40 mg</h6>
+        </div>
+        <div class="card-row">
+          <span class="light float-right">0 tablets</span>
+          <h6>Telmisartan 80 mg</h6>
+        </div>
+      </div>
+
+      <div class="card">
+        <h3 style="margin-bottom: 8px;">Diuretic tablets</h3>
+        <span class="text-orange">35 days of drug stock</span>
+
+        <div class="card-row">
+          <span class="light float-right">19 tablets</span>
+          <h6>Chlorthalidone 12.5 mg</h6>
+        </div>
+        <div class="card-row">
+          <span class="light float-right">200 tablets</span>
+          <h6>Chlorthalidone 25 mg</h6>
+        </div>
+      </div>
+
+      <div class="button-fixed-bottom">
+        <button onclick="closeWindow('drug-stock-submitted', 'progress-start'); return false" class="button">DONE</button>
+      </div>
+
+  </div>
+</div>
```

Comment:
I'm not sure what to do w/ the above static content -- it is showing the current state of drug stock, but I think its better put into a new view that the android app can navigate to. I'll add it to the open questions in the story.